include/dg/llvm/LLVMDependenceGraphBuilder.h

```diff
@@ -73,6 +78,7 @@ class LLVMDependenceGraphBuilder {
     std::unique_ptr<LLVMPointerAnalysis> _PTA{};
     std::unique_ptr<LLVMReachingDefinitions> _RD{};
     std::unique_ptr<LLVMDependenceGraph> _dg{};
+    std::unique_ptr<ControlFlowGraph> _controlFlowGraph{};
```

Comment:
This is the threaded CFG, right? Maybe the name should reflect that (e.g. something like `_threadedControlFlowGraph` or `threadedCFG`.